ux.symfony.com/templates/main/homepage.html.twig

```diff
@@ -71,7 +71,7 @@
         </div>
         <div class="col-12 col-md-6  mt-3 mt-md-0">
             <twig:CodeBlock
-                filename="assets/controllers/markdown-controller.js"
+                filename="assets/controllers/markdown_controller.js"
```

Comment:
I think today we used "-controller" as naming convention..

Not sure we should change for only one :|

Reply:
What about renaming then the hello-controller?

Reply:
If you think that'll improve the coherence yes for sure!